server/chat/background/task.py

```
@@ -1319,6 +1329,40 @@ async def _execute_background_chat(
+
+            _update_incident_status_and_aurora(incident_id, "analyzed", "summarizing", user_id=user_id)
+
+            try:
+                from chat.background.summarization import generate_incident_summary_from_chat
+                generate_incident_summary_from_chat.delay(
+                    incident_id=incident_id,
+                    user_id=user_id,
+                    session_id=session_id,
+                )
+            except Exception as e:
+                logger.error(f"[BackgroundChat] Failed to enqueue post-RCA summarization: {e}")
```

Ruff (0.15.13)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)